src/LintPlugin.php

```diff
@@ -37,7 +37,8 @@ public function activate(Composer $composer, IOInterface $io)
     {
         $this->composer = $composer;
         $this->io = $io;
-        $this->linter = new Linter();
+        $config = $this->composer->getConfig()->get('sllh-composer-lint');
+        $this->linter = new Linter($config ?: array());
```

Comment:
[OTVs can be used to breakup long lines and complicated expressions](https://a-nickels-worth.blogspot.fr/2016/04/a-guide-to-naming-variables.html#orgheadline13), so this one may be ok.